README.md

```diff
@@ -1,10 +1,46 @@
 # React Native Set Version
 
-React Native Set Version is a tool that let you set your version easily trough your react-native application. It will modify the following files if found:
+React Native Set Version is a tool that lets you set your version easily trough your react-native application. It will modify the following files if found:
```

Comment:
I see multiple places where the English could be made more clear in your README. Would you mind if I made the changes directly instead of adding lots of comments?

Reply:
Sure go ahead :smiley: